charts/gardener/gardenlet/charts/runtime/templates/priorityclass.yaml

```diff
@@ -1,7 +0,0 @@
-apiVersion: {{ include "priorityclassversion" . }}
-kind: PriorityClass
-metadata:
-  name: gardenlet
```

Comment:
Who deletes this old `PriorityClass`? It should not remain orphaned in the system.

Reply:
What do you think should be the best approach to clean up the old PriorityClasses that we are replacing now?

Reply:
When the gardenlet comes up it could delete the old class